samples_project/Assets/SampleViewer/Samples/Geocoding/readme.md

```diff
@@ -1,19 +1,29 @@
-# Geocoding
+# Find places or search for an address
```

Comment:
```suggestion
# Geocoding and reverse geocoding
```

Reply:
We better have the top two bold titles as much as possible because many of the required tutorials are covered by samples in GE.
https://docs.afd.geocloud.com/doc-authors/tutorials/names-and-descriptions/#geocoding-and-search

Reply:
so which one do you intend to use? `Find places or search for an address`?

Reply:
yeap, because we have both in one sample. At least that's my idea.

Reply:
why not using geocoding and reverse geocoding?